Dockerfiles/centos8

```diff
@@ -0,0 +1,31 @@
+FROM centos:8
+
+ENV OSCAP_USERNAME oscap
+ENV OSCAP_DIR content
+ENV BUILD_JOBS 4
+
+RUN true \
+  && dnf -y upgrade \
+  && dnf -y install cmake openscap-utils python3-jinja2 python3-pip python3-pyyaml \
+  && dnf -y --enablerepo=PowerTools install ninja-build \
+  && pip3 install argparse \
```

Comment:
Isn't argparse a part of python3-libs? Is there any reason to install it by pip?

Reply:
So true, just couldn't find `python3-libs` earlier for some reason, `python3-pip` is however still required.